Tribler/Main/tribler_main.py

```diff
@@ -112,7 +114,7 @@ def __init__(self, params, installdir, autoload_discovery=True,
 
         self.webUI = None
         self.utility = None
-
+        self.boosting_manager = None
```

Comment:
I'm not confident with the boosting manager yet but I guess the creation and setup of this manager can be moved to `LaunchManyCore`? If so, it will make my work when I will remove the wx GUI a lot easier :)